### DISCUSSION TITLE
The tooltip for area chart not showing properly with invert function

### DISCUSSION THREAD

**Discussion by renjiie** [airbnb/visx#1546] [Q&A]
Hi Team, you have done a great work with Visx library. Thanks!

Codesandbox link : https://codesandbox.io/s/dawn-wind-unish3?file=/Example.tsx

In this example I am trying to render an area chart with tooltip and axis. Tooltip is rendering but it is doing some calculations to render the closest value and it is not showing up properly. Also the left axis is not showing up.

**Comment by williaster** [comment]
converting this to a discussion because it's a usage problem not a visx issue. 

short on time but here's an updated sandbox
- you need to account for margin.left properly and offset the left axis `margin.left` else it is clipped by the svg (svg doesn't flow like html, it's pixel based)
- it's very difficult to debug tooltips by logging to the console so I made the tooltip snap to the data point your bisect logic found. this enabled me to find that your array of data points was not sorted by time which is an assumption outlined in the d3 bisect docs. fixing the sort fixed the tooltip

https://codesandbox.io/s/focused-noether-j5i9q4
